web/api/v6/report_server.thrift

```diff
@@ -93,6 +93,11 @@ enum ExtendedReportDataType {
   FIXIT = 20,
 }
 
+enum CommentKind {
```

Comment:
I'm thinking maybe it would be better if we introduce some prefix for the thrift types.
There are conversion functions which convert from CommentKind to CommentKind one of them is a thrift enum and the other is a local enum it is hard to follow which type is used where and what is converted to what.